appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/math/Vector.kt

```diff
@@ -37,6 +38,7 @@ fun angleDegrees(vector: Offset): Float  {
     return (deg.toFloat() + 90) % 360
 }
 
+@Suppress("MagicNumber")
```

Comment:
Maybe it's better to add it on the entire file or to not turn it on at all? I don't find too much value in adding a restriction we need to keep suppressing everywhere.

Reply:
I think we should only add it to the top of files we don't plan to fix.

I think some of these should be revisited, and constants created. Thoughts?

Reply:
I'm not convinced it adds value in most cases. It also won't play well with `MotionController` instances where all numbers are ui properties. 

I'd rather remove the rule (at least in the scope of interactions/components, we can consider keeping it in places we don't expect ui properties to pop up) as it just adds friction 99% of the time, and an unjustifiable magic number is one of the easier things to catch on a code review anyway.